woocommerce/woocommerce-android
# [Woo POS][Cash & Receipts] Add optional take payment button and empty state for cash

Closes: #13004 


### Description

The PR adds optional "take cash payment" button that open a mock cash payment screen

### Testing information

The most important at this stage to validate that in the release version the take payment cash button is not visible and doesn't bring any regression to the production POS flow

Other than that please validate:
* More -> POS
* Add a product 
* Checkoung -> notice a new cash payment button
* Notice that it opens the new UI

### Images/gif


https://github.com/user-attachments/assets/7787ff86-62d0-4de8-9e93-ae6bd92e2d8d



## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
